packages/fluentui/react-northstar/src/components/Chat/ChatItem.tsx

```diff
@@ -61,19 +62,27 @@ export type ChatItemStylesProps = Pick<ChatItemProps, 'attached' | 'contentPosit
 /**
  * A ChatItem is container for single entity in Chat (e.g. message, notification, etc).
  */
-export const ChatItem = (React.forwardRef<HTMLLIElement, ChatItemProps>((props, ref) => {
+export const ChatItem = (React.forwardRef<HTMLLIElement, ChatItemProps>((inputProps, ref) => {
   const context = useFluentContext();
   const { setStart, setEnd } = useTelemetry(ChatItem.displayName, context.telemetry);
   setStart();
 
-  const chatDensity = useChatDensityContext();
+  const chatProps = (useContextSelectors(ChatContext, {
+    density: v => v.density,
+    accessibility: v => v.behaviors.item,
+  }) as unknown) as ChatContextSubscribedValue;
```

Comment:
you could wrap this in `useChatContext` and reuse that in `ChatItem` and `ChatMessage`

Reply:
actually it's not that bad, the `unknown` case sneaking its way around is not nice tho :/

Reply:
you could potentially wrap `useContextSelector` with `useChatContext` so the unknown case happens once inside the `chatContext.ts` module, it's limited to one place and it's where the context is defined, so it's nicer IMO.

```ts
useChatContext(v => ({ density: v => v.density }))
```